webpack/components/PaginationRow/index.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import { Paginator } from 'patternfly-react';
 import { isEqual } from 'lodash';
 
-const defaultPerPageOptions = [5, 10, 15, 25, 50];
+const defaultPerPageOptions = [5, 10, 15, 20, 25, 50];
```

Comment:
We should probably add whatever the default value in the database to this list as well.  See https://projects.theforeman.org/issues/24477.